pkg/metrics/grafana/tidb.json

```diff
@@ -16487,7 +16487,7 @@
           "targets": [
             {
               "exemplar": true,
-              "expr": "sum(tidb_disttask_subtasks{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", status=~\"pending|running|reverting|revert_pending|paused\"}) by (exec_id)",
+              "expr": "sum(tidb_disttask_subtasks{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", status=~\"pending|running|failed|canceled|paused\"}) by (exec_id)",
```

Comment:
I think "failed" and "canceled" should not be considered as running?

Reply:
The panel is `Uncompleted Subtask Distribution on TiDB Nodes`, so i think the except for 'succeed' tasks, all status should be included ?